frame/contracts/src/wasm/mod.rs

```diff
@@ -543,8 +545,8 @@ mod tests {
 			}
 			Ok(result)
 		}
-		fn caller(&self) -> &AccountIdOf<Self::T> {
-			&ALICE
+		fn caller(&self) -> Caller<Self::T> {
```

Comment:
curious why you have decided to add the field vs just return `Caller::Account(ALICE)` ?